### ISSUE TITLE
Can't escape out of blockquotes with return

### ISSUE THREAD

**Issue by mattkenefick** [tiff/wysihtml5#274]
Often in WYSIWYG editors, if you're in a blockquote, you can return twice to escape it similar to how you'd escape a list item. Doesn't seem to take effect in this. 

Anyone else experience this?

**Comment by zrod** [comment]
Yes, not only with blockquotes but with pre and code as well (actually with any multi-line tag).

**Comment by mattkenefick** [comment]
Anyone have clever resolutions?

**Comment by theseanz** [comment]
Anyone have any luck with this? I'm struggling with it right now.

**Comment by mattkenefick** [comment]
I added a few commits to this forked branch a little while ago. https://github.com/percolate/wysihtml5/commits/master

It used RegEx to figure out if it should let you out. As I recall, it worked pretty well across browsers. Give it a shot.
